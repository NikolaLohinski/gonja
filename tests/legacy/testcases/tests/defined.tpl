{{ not_found is defined }}
{{ not_found is not defined }}
{{ simple is defined }}
{{ simple is not defined }}
{{ simple.str is defined }}
{{ simple.str is not defined }}
{{ simple.missing is defined }}
{{ simple.missing is not defined }}
