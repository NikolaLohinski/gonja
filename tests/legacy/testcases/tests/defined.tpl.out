False
True
True
False
True
False
False
True
